### ISSUE TITLE
Cannot Read property 'constructor'

### ISSUE THREAD

**Issue by WallyThePenguin** [dirigeants/klasa#410]
### Describe the issue
Bot Won't Startup because of this error(below), i deleted and redownloaded the node-module to get up-to-date with my bot and all it needs is d.js and klasa. I can't get back to Discord server cause im supposedly banned for some reason, Waleed#8808 is my code if you wanna look up why (tbh was a long time so forgot) I tried to maybe "undo" it by redoing the steps that i have done, nothing.
### Code or steps to reproduce
deleted and redownload node-module
```js
Uncaught Promise Error:
TypeError: Cannot read property 'constructor' of undefined
    at SettingsCache.get defaultDataSchema [as defaultDataSchema] (C:\Users\PC User\Desktop\Ioch-master\node_modules\klasa\src\lib\settings\SettingsCache.js:90:38)
    at new SettingsCache (C:\Users\PC User\Desktop\Ioch-master\node_modules\klasa\src\lib\settings\SettingsCache.js:30:98)
    at KlasaClient.login (C:\Users\PC User\Desktop\Ioch-master\node_modules\klasa\src\lib\Client.js:366:19)
    at <anonymous>
```

### Further details

- **discord.js version**:11.4.2
- **node.js version**:v8.9.4
- **Klasa version**:0.4.0
- [ ] I have modified core files.
- [ ] I have tested the issue on latest master. Commit hash:

**Comment by kyranet** [comment]
Two things:

1. The stable branch (published in npm) may not be "correctly" functional, it's pretty much stale and discord.js merged several breaking changes that break `klasa@0.4.0`. Support for everything related to SettingsGateway v1.x (what this version has) is pretty limited, due to the amount of issues found during the development of `klasa@0.5.0-dev`.
1. klasa has always required `discord.js@12.0.0`, specially with `klasa@0.5.0-dev` since it uses structures. You should also consider using master and not stable due to the huge overall improvements, specially related to performance and RAM usage. Even if SettingsCache had worked for you, you'd soon find issues since we rely on `UserStore#fetch` (discord.js master), which is not available in `discord.js@11.x` (`Client#fetchUser`), between many other issues.

tl;dr: You should use master, stable is full of already-fixed bugs.
